src/main/webapp/js/statusMessage.js

```diff
@@ -1,3 +1,14 @@
 $(document).ready(function() {
-	document.getElementById('statusMessage').scrollIntoView();
+    var statusMessage = document.getElementById('statusMessage');
+    var navbarHeight = '70';
+
+    // scroll to element
+    statusMessage.scrollIntoView(true);
+
+    // now account for fixed header
+    var scrolledY = window.scrollY;
+
+    if (scrolledY) {
+      window.scroll(0, scrolledY - navbarHeight);
```

Comment:
Perhaps instead of scrolling twice, just compute the final height and scroll once?